core/src/main/java/bisq/core/user/PreferencesPayload.java

```diff
@@ -47,6 +47,8 @@
 
 import javax.annotation.Nullable;
 
+import static bisq.core.btc.wallet.Restrictions.getDefaultBuyerSecurityDepositAsPercent;
```

Comment:
More static imports, in case you meant to revert them all, otherwise it's ok, just feels a bit excessive for these kinds of use cases.

Reply:
Here I left it by intent to reduce the line length for L125 & L127